packages/styles/scss/components/_multilinkcard.scss

```diff
@@ -125,6 +125,14 @@
                 display: none;
               }
             }
+
+            &#{$self}--no-image {
+              --max-width: #{px-to-rem(920px)};
```

Comment:
@Shashika6 what is this for? Why should there being an image or not affect the width of the card? Also, it doesn't seem to have any effect.

Reply:
When there is no image, the component has a max-width of 920px. 

There was a requirement by design.